doc/api_base.rst

```diff
@@ -78,9 +78,9 @@
       base.repos.add(repo)
       base.fill_sack()
 
-  .. method:: do_transaction([display])
+  .. method:: do_transaction([displays])
```

Comment:
I just realized that this is wrong. We need both `display` and `displays` here because `do_transaction(foo)` would pass `foo` as `display` not as `displays`.